flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
why?

Reply:
Because the binary file used by the app is x86_64. There is also an arm version but I don't know how to get the pc architecture and use the correct binary

Reply:
so you are not building from source? you should be

Reply:
`$FLATPAK_ARCH` will be set to the architecture being build. (and coincidentally being run)